base/doc/ltnews41.tex

```diff
@@ -311,6 +311,18 @@ \section{News from Tagged PDF project}
 
 \emph{to write}
 
+\subsection{Setting up math tagging}
+
+With the \LuaTeX{} engine there are now various options for accessible math described in full details in latex-lab-math.pdf. To simplify the setup a new
+key \texttt{math/setup} can be used that  accepts a comma list with the values \texttt{structelem} (add MathML structure element), \texttt{mathml-AF}
+(attach MathML associated file) or \texttt{tex-AF} (attach the TeX source).
```

Comment:
I still think that  mathml-SE, mathml-AF, tex-AF would be better, but I don't havee very strong feelings on that --- any votes from others?

Reply:
I suspect that I am not being asked.  Who knows?

Reply:
Any decision here? @FrankMittelbach @davidcarlisle ?

Reply:
I already said that I think mathml-SE is better imho, and I asked for opinions of others. what else could I say? :-) @josephwright ?

Reply:
> I suspect that I am not being asked. Who knows?

you have been asked initially as everybody else, but you have been basicaly away the whole week from any dicussion or review ask, so I was expecting you you be offline and busy with other things

Reply:
I was not!!

Neither initially, nor in this conversation!
(Only two requests just above here!)

Why am I never, ever asked by some of you!